src/main/java/com/faforever/api/config/error/ErrorResult.java

```diff
@@ -1,9 +1,32 @@
 package com.faforever.api.config.error;
 
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
 import lombok.Data;
+import lombok.RequiredArgsConstructor;
+
+import java.util.Map;
 
 @Data
+@RequiredArgsConstructor
+@AllArgsConstructor
+@JsonInclude(JsonInclude.Include.NON_NULL)
 public class ErrorResult {
+  @JsonProperty("status")
+  private final Integer httpStatusCode;
+  @JsonProperty("code")
+  private Integer appCode;
```

Comment:
from the rest/jsonapi point of view I thought httpstatus is necessary, but since only our custom errors have appcode that those should be optional. Or do we want to group framework exceptions to some error codes?